src/components/panes/EditCampaignPane.jsx

```diff
@@ -41,6 +42,17 @@ export default class EditCampaignPane extends PaneBase {
         }
     }
 
+    getPaneSubTitle(data) {
+        if (this.props.campaignItem && !this.props.campaignItem.isPending) {
+            const link= 'https://zetk.in/o/'+this.props.activeOrg.id+'/campains/'+this.props.campaignItem.data.id;
+
+            return <a href={link} target="_blank"> {link}</a>;
```

Comment:
There is a redundant whitespace on this line before the second `{link}` which should be removed.